rocket/src/main/scala/craft/ShiftRegisterMem.scala

```diff
@@ -3,7 +3,7 @@
 package craft
 
 import chisel3._
-import chisel3.core.requireIsHardware
+import chisel3.internal.requireIsHardware
```

Comment:
Shouldn't this be `import chisel3.experimental.requireIsHardware`?

Reply:
Should it? This is what the deprecation warning chisel3.core gave me. I'll switch if experimental is preferred

Reply:
Hmm. Maybe I'm out of step. The `experimental` piece may be a compatibility bone. If the warning says to use `internal`, then I'd use it.